docs/Documentation/07-publish-via-ci.md

```diff
@@ -0,0 +1,33 @@
+# Publish via Continuous Integration
+
+The "Publish" button in Studio Web is a quick and convenient way to publish changes (e.g.: updating documentation), but if you are working with a whole team of people, it can become a bit of a bother. Maybe they commit some changes to master and forget to publish them after their pull request is merged.
+
+To handle this, we have a handy CLI package which can automate publishing changes happening via Git.
+
+1. Install the `@stoplight/cli` package from NPM 
```

Comment:
@lottamus Could we mention https://github.com/stoplightio/circleci-orbs#stoplightcli- on here to simplify things further? 

> If you use CircleCI, do this...